Description: Do not strip during install.
 Any stription must be left to package building tools.  Only telnetlogin
 was left unattended, since telnet and telnetd ware fixed before package
 0.17.24+0.1-7.1.

Author: Mats Erik Andersson <debian@gisladisker.se>
Forwarded: no
Last-Update: 2015-03-07

--- netkit-telnet-ssl-0.17.24+0.2/telnetlogin/Makefile.orig
+++ netkit-telnet-ssl-0.17.24+0.2/telnetlogin/Makefile
@@ -14,7 +14,7 @@
 $(OBJS): ../version.h
 
 install: telnetlogin
-	install -s -m4750 -oroot -gtelnetd telnetlogin $(INSTALLROOT)$(SBINDIR)
+	install -m4750 -oroot -gtelnetd telnetlogin $(INSTALLROOT)$(SBINDIR)
 	install -m$(MANMODE) telnetlogin.8 $(INSTALLROOT)$(MANDIR)/man8
 
 clean:
